cvat/apps/documentation/installation_automatic_annotation.md

```diff
@@ -0,0 +1,84 @@
+
+### Semi-automatic and automatic annotation
+
+- To bring up cvat with auto annotation tool, **do not use** `docker-compose up`.If you did first make sure all containers are stopped `docker-compose down`
+
+
+  From cvat root directory, you need to run:
+  ```bash
+  docker-compose -f docker-compose.yml -f components/serverless/docker-compose.serverless.yml up -d
+  ```
+  If you did any changes to the docker-compose files, make sure to add `--build` at the end.
+
+  To stop the containers, simply run:
+
+  ```bash
+  docker-compose -f docker-compose.yml -f components/serverless/docker-compose.serverless.yml down
+  ```
+
+
+- You have to install `nuctl` command line tool to build and deploy serverless
+  functions. Download [version 1.5.8](https://github.com/nuclio/nuclio/releases).
+  It is important that the version you download matches the version in
+  [docker-compose.serverless.yml](/components/serverless/docker-compose.serverless.yml)
+  when you downloaded the nuclio give them proper permission and do a softlink
+  ```
+  sudo chmod +x nuctl-<version>-linux-amd64
+  sudo ln -sf $(pwd)/nuctl-<version>-linux-amd64 /usr/local/bin/nuctl
+  ```
+
+- Create `cvat` project inside nuclio dashboard where you will deploy new
+  serverless functions and deploy a couple of DL models. Commands below should
+  be run only after CVAT has been installed using docker-compose because it
+  runs nuclio dashboard which manages all serverless functions.
+
+  ```bash
+  nuctl create project cvat
+  ```
+
+  ```bash
+  nuctl deploy --project-name cvat \
+    --path serverless/openvino/dextr/nuclio \
+    --volume `pwd`/serverless/openvino/common:/opt/nuclio/common \
+    --platform local
+  ```
+
+  ```bash
+  nuctl deploy --project-name cvat \
+    --path serverless/openvino/omz/public/yolo-v3-tf/nuclio \
+    --volume `pwd`/serverless/openvino/common:/opt/nuclio/common \
+    --platform local
+  ```
+
+
+  If your function is running on GPU, you should add `--resource-limit nvidia.com/gpu=1` to the above command or, alternatively, add gpu resources dircetly into the function.yaml see [tensorflow-fast-rcnn-gpu](../../../serverless/tensorflow/
```

Comment:
Also it is necessary to install nvidia support for the docker. Probably need to mention something like that as TF does https://www.tensorflow.org/install/docker#gpu_support

Reply:
Yes, they need to have the Nvidia-toolkit. I will add that to the documentation.

Reply:
- [ ] add docker instructions for gpu support to documentation